applications/matter_bridge/CMakeLists.txt

```diff
@@ -60,6 +60,20 @@ if(CONFIG_BRIDGE_ONOFF_LIGHT_BRIDGED_DEVICE)
     )
 endif() # CONFIG_BRIDGE_ONOFF_LIGHT_BRIDGED_DEVICE
 
+if(CONFIG_BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE)
+    target_sources(app PRIVATE
+        src/bridged_device_types/humidity_sensor.cpp
+        src/bridged_device_types/ble_environmental_data_provider.cpp
+    )
+endif() # CONFIG_BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE
+
+if(CONFIG_BRIDGE_TEMPERATURE_SENSOR_BRIDGED_DEVICE)
+    target_sources(app PRIVATE
+        src/bridged_device_types/temperature_sensor.cpp
+        src/bridged_device_types/ble_environmental_data_provider.cpp
```

Comment:
Is it ok to be added it twice when both temperature and humidity sensors are enabled?

Reply:
I guess it doesn't hurt, but we'd better avoid that anyway.